handbook/engineering/README.md

```diff
@@ -669,25 +672,28 @@ After it's in a release formally, the bug should be treated like any other piece
 #### Fast track for Fleeties
 Fleeties do not have to wait for QA to reproduce the bug. If you're confident it's reproducible, it's a bug, and the reproduction steps are well-documented, it can be moved directly to the reproduced state.
 
-#### Release testing
-When a release is in testing, QA should use the the Slack channel #help-qa to keep everyone aware of issues found. All bugs found should be reported in the channel after creating the bug first.
+### Release testing
+When a release is in testing, QA should use the Slack channel #help-qa to keep everyone aware of issues found. All bugs found should be reported in the channel after creating the bug first.
 
-In the #help-qa channel, product may decide whether the bug is a release blocker. When a release-blocking bug is found, product is responsible for communicating the delay to company stakeholders. 
+In the #help-qa channel, product may decide whether the bug is a critical or release-blocking. When a critical bug is found, product is responsible for following the [critical bug notification process](https://fleetdm.com/docs/contributing/releasing-fleet#critical-bug-notification-process) below. When a release-blocking bug is found, product is responsible for communicating any delays to company stakeholders. 
 
-Release blockers include:
-1. Critical bugs (defined below)
-2. New functionality that impacts previous stable functionality
-3. Incomplete features as defined or implied in the specs
+Release blockers must be fixed before a release can be cut. Non-release-blocking bugs may be addressed during a subsequent release per the standard [bug process](https://fleetdm.com/docs/contributing/releasing-fleet#bug-process). 
 
-Release blockers must be fixed before a release can be cut. Non-release-blocking bugs may be addressed during a subsequent release per the standard bug process (defined above).
+### Release-blocking bugs
+A release-blocking bug is identified by the `~release blocker` label. It becomes a release-blocking bug if:
+* New functionality that impacts previous stable functionality
+* Incomplete features as defined or implied in the specs
+* It does not meet the criteria of a critical bug
+* We will delay the release to ensure a fix if necessary
 
-#### Critical bugs
-A critical bug is defined as behavior that: 
+### Critical bugs
+A critical bug is a bug with the `~critical bug` label. A critical bug is defined as behavior that: 
 * Blocks the normal use a workflow
 * Prevents upgrades to Fleet
 * Causes irreversible damage, such as data loss
 * Introduces a security vulnerability
 
+#### Critical bug notification process
 We need to inform customers and the community about critical bugs immediately so they don’t trigger it themselves. When a bug meeting the definition of critical is found, the bug finder is responsible for raising an alarm.
```

Comment:
not relevant to this PR exactly, but I don't think we've been following this process.

Reply:
Agreed, I'll ensure we do next time we need to issue a patch.